src/source/Signaling/LwsApiCalls.c

```diff
@@ -643,6 +643,10 @@ STATUS lwsCompleteSync(PLwsCallInfo pCallInfo)
     return retStatus;
 }
 
+BOOL isCallResultFailureRetryable(PCallInfo pCallInfo) {
+    return (STRNSTR(pCallInfo->responseData, "Signature expired", pCallInfo->responseDataLen) == NULL);
```

Comment:
Is "Signature expired" the only scenario where we would want to bail out? As a long term plan, we might want to translate these strings to status codes and create a list of retriable vs non retriable errors if that is not already happening. What do you think?

Reply:
Agreed, I expect that to come in future PRs.